pkg/stub/handler.go

```diff
@@ -154,7 +158,7 @@ func (h *Handler) SpecValidation(srcfg *v1alpha1.SamplesResource) error {
 		return nil
 	}
 	cm, err := h.configmapclientwrapper.Get(h.namespace, v1alpha1.SamplesResourceName)
-	if cm == nil || err != nil {
+	if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
if it is an IsNotFound err, i think you need to make sure cm gets set to a non-nil value.  I wouldn't count on get always returning an empty one (i find it weird that it ever does)

Reply:
also i'm not sure what you expect to do if it is an IsNotFoundErr?

Reply:
Yeah need to error out with a ConditionUnknown status for the ConfigurationValid condition ... presumably somebody deleted the configmap underneath us, or some such pre-existing problem